### PR TITLE
[12.0][FIX] stock_move_location: Keep product lines when user launchs wizard from stock quant view

### PR DESCRIPTION
**Pull request by sergio-teruel**
cc @Tecnativa TT21885

### CODE REVIEW

**Review by pedrobaeza — APPROVED** [review]

**Comment by sergio-teruel** [comment]
Added new commit --- @carlosdauden @pedrobaeza review please

File: stock_move_location/__manifest__.py [hunk 1]
```diff
@@ -4,7 +4,7 @@
 
 {
     "name": "Move Stock Location",
-    "version": "12.0.1.2.0",
+    "version": "12.0.1.3.0",
```
Comment (pedrobaeza) [inline stock_move_location/__manifest__.py:7]: Remember that there's no need to change this manually.

File: stock_move_location/migrations/12.0.1.3.0/pre-migrate.py [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+from openupgradelib import openupgrade
+
+
+@openupgrade.migrate()
+def migrate(env, version):
+    """
+    Remove old table relation
+    """
+    sql = """
+        DROP TABLE IF EXISTS
```
Comment (pedrobaeza) [inline stock_move_location/migrations/12.0.1.3.0/pre-migrate.py]: Why do you need to do this?

File: stock_move_location/wizard/stock_move_location.py [hunk 3]
```diff
@@ -42,6 +42,10 @@ def _get_default_picking_type_id(self):
     stock_move_location_line_ids = fields.Many2many(
         string="Move Location lines",
         comodel_name="wiz.stock.move.location.line",
+        relation="wiz_stock_move_location_wiz_stock_move_location_line_rel",
```
Comment (pedrobaeza) [inline stock_move_location/wizard/stock_move_location.py]: And why this is needed to be specify?

File: stock_move_location/wizard/stock_move_location.xml [hunk 4]
```diff
@@ -13,6 +13,12 @@
                 Edit Locations
               </label>
               <field name="edit_locations" widget="boolean_toggle"/>
+          </div>
+           <div groups="stock.group_stock_multi_locations">
+              <label for="apply_putaway_strategy">
+                Apply putaway strategy for move products
```
Comment (pedrobaeza) [inline stock_move_location/wizard/stock_move_location.xml]:
```suggestion
                Apply putaway strategy for moving products
```

**Comment by sergio-teruel** [comment]
Changes done

**Comment by pedrobaeza** [comment]
Please check Travis

**Review by rafaelbn — APPROVED** [review]
:+1: 

Please @sergio-teruel review travis :red_circle:

**Comment by sergio-teruel** [comment]
Travis OK

**Review by carlosdauden — APPROVED** [review]

File: stock_move_location/wizard/stock_move_location.py [hunk 5]
```diff
@@ -42,6 +42,9 @@ def _get_default_picking_type_id(self):
     stock_move_location_line_ids = fields.Many2many(
         string="Move Location lines",
         comodel_name="wiz.stock.move.location.line",
+        column1="move_location_wiz_id",
+        column2="move_location_line_wiz_id",
+
```
Comment (MiquelRForgeFlow) [inline stock_move_location/wizard/stock_move_location.py:47]:
```suggestion
```

😏

**Pull request merged**